### PR TITLE
manifest: update for nrfxlib PR 456

### PR DESCRIPTION
**Pull request by greg-fer**
Updated manifest in west.yml.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -99,7 +99,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: fb936182435a1ae90fafbb8840e70a45d6f99a4e
+      revision: pull/456/head
```
Comment (rlubos) [inline west.yml:102]: Please update SHA: `121c6fa46dd7f6ee9852751da0357a41ba182a1e`

Comment (greg-fer) [reply west.yml]: Done

**Review by rlubos — APPROVED** [review]

**Comment by pdunaj** [comment]
Hi, is there a chance this will be merged this week? I don't want to duplicate the manifest PRs to not make the CI situation worse.

**Comment by pdunaj** [comment]
@greg-fer can you retrigger tests?

**Comment by greg-fer** [comment]
> @greg-fer can you retrigger tests?

How can I do this?

**Comment by pdunaj** [comment]
nfc/homekit/boot test failed. I guess the rebase is needed

**Comment by pdunaj** [comment]
@greg-fer I have rebased your branch

File: west.yml [hunk 2]
```diff
@@ -99,7 +99,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: fb936182435a1ae90fafbb8840e70a45d6f99a4e
+      revision: 121c6fa46dd7f6ee9852751da0357a41ba182a1e
```
Comment (tejlmand) [inline west.yml:102]: This is no longer head of master in nrfxlib.
Superseded by: https://github.com/nrfconnect/sdk-nrf/pull/4354

**Review by tejlmand — CHANGES_REQUESTED** [review]

File: west.yml (hunk 2 above)
Comment (pdunaj) [reply west.yml:102]: Thanks! I will not rebase that nor my original PR.

@greg-fer This one can be closed when #4354 gets in. There is no point in duplicating tests.

**Comment by pdunaj** [comment]
Not needed anymore

**Pull request closed without merging**